tools/diffbind/diffbind.xml

```diff
@@ -3,6 +3,7 @@
     <requirements>
         <requirement type="binary">Rscript</requirement>
         <requirement type="R-module">DESeq2</requirement>
+        <requirement type="package" version="1.2.8">zlib</requirement>
```

Comment:
Is this really needed? Maybe just adding:
```
    <repository name="package_zlib_1_2_8" owner="iuc">
        <package name="zlib" version="1.2.8" />
    </repository>
```
to `tool_dependencies.xml` (as for libxml2 in https://github.com/galaxyproject/tools-iuc/commit/8ed92ab064fd3d7e19530b5ff0bd6da0bcef8ab0 ) would be better.